compiler/src/iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp

```diff
@@ -43,19 +43,46 @@ using mlir::iree_compiler::IREE::LinalgExt::LinalgVectorLoweringOptions;
 // IREE specific functions
 //===----------------------------------------------------------------------===//
 
-/// Returns the op that contains lowering config. Returns failure if there are
-/// multiple op having lowering config.
-static FailureOr<Operation *> getRootOp(func::FuncOp funcOp) {
+/// Returns the op that contains lowering config. Checks whether the provided op
+/// contains the lowering config and returns it. Otherwise, tries to find the
+/// lowering config across the function. If there are multiple ops with the same
+/// lowering configs, returns the first one found. Returns failure if there are
+/// multiple op with different lowering config.
```

Comment:
There is a method on SPIR-V / CUDA side that does something else....  Ill try to find it

Reply:
I think @nicolasvasilache introduced this for the transform dialect. Happy to remove this when I remove the dependency with lowering_config (WIP).